main.go

```diff
@@ -23,6 +26,21 @@ var (
 )
 
 func main() {
+
+	signalChannel := make(chan os.Signal, 2)
+	signal.Notify(signalChannel, os.Interrupt, syscall.SIGTERM)
+	go func() {
+		sig := <-signalChannel
+		switch sig {
+		case syscall.SIGTERM:
+			//handle SIGTERM
```

Comment:
I'd maybe remove this comment as this is quite obvious from `case syscall.SIGTERM` above :slightly_smiling_face: